src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -3556,9 +3556,13 @@ public sealed partial class Version : System.IComparable, System.IComparable<Sys
         public static bool operator <(System.Version v1, System.Version v2) { throw null; }
         public static bool operator <=(System.Version v1, System.Version v2) { throw null; }
         public static System.Version Parse(string input) { throw null; }
+        public static System.Version Parse(ReadOnlySpan<char> input) { throw null; }
         public override string ToString() { throw null; }
         public string ToString(int fieldCount) { throw null; }
+        public bool TryFormat(System.Span<char> destination, out int charsWritten) { throw null; }
```

Comment:
Span is available in all of these, and we're ensuring that all of the new span-based APIs we're adding are added to both coreclr and corert.